frontend/ui/widget/spinwidget.lua

```diff
@@ -111,21 +109,15 @@ function SpinWidget:update(numberpicker_value, numberpicker_value_index)
         value_widget,
     }
 
-    local value_title = FrameContainer:new{
-        padding = Size.padding.default,
-        margin = Size.margin.title,
-        bordersize = 0,
-        TextWidget:new{
-            text = self.title_text,
-            max_width = self.width - 2 * (Size.padding.default + Size.margin.title),
-            face = self.title_face,
-        },
-    }
-    local value_line = LineWidget:new{
-        dimen = Geom:new{
-            w = self.width,
-            h = Size.line.thick,
-        }
+    local title_bar = TitleBar:new{
+        width = self.width,
+        align = "left",
+        with_bottom_line = true,
+        title = self.title_text,
+        title_face = self.title_face,
```

Comment:
Does it make sense to keep allowing these widgets' title_face to be customizable ? Do we have some instances using a different font ? Shouldn't we prevent that to enforce consistency ?